examples/LoRaDuplex/LoRaDuplex.ino

```diff
@@ -0,0 +1,101 @@
+/*
+  LoRa Duplex communication
```

Comment:
The test example was just to show the work on the sync word issue. I think it's too complex for an example, you can delete that whole folder when you're done with it.